customer/src/main/resources/application.yml

```diff
@@ -0,0 +1,3 @@
+spring:
+  hateoas:
+    use-hal-as-default-json-media-type: false
```

Comment:
was tut dieser Parameter?

Reply:
stellt HAL links aus (mit Param: `links` - ohne Param `_inks`, geht auch komplexer...). 
Details:
Nichts gegen den HAL Standard, aber die Link Spezifikation finde ich nicht schön. Außerdem verhält sich das bei Spring HATEOAS tw. komisch. Die Links sind nicht immer als `_links` Objekt im json abgbildet. Hab nicht ganz verstanden warum das manchmal als `links` und manchmal als `links` abgenildet wird - finde das verwirrend. Bin nicht sicher woran das liegt, müsste mich wohl dafür in den Standard einlesen (HAL Link Standard sieht dies vor? Muss noch irgendwie gewrapped werden?).
HAL Standard ist ziemlich komplex, würde ich den Einsteigern jetzt nicht gleich zumuten (vielleicht in einer separaten Stage...), details: http://stateless.co/hal_specification.html